Sources/Host/Darwin/PTrace.cpp

```diff
@@ -57,16 +58,8 @@ ErrorCode PTrace::kill(ProcessThreadId const &ptid, int signal) {
 ErrorCode PTrace::readString(ProcessThreadId const &ptid,
                              Address const &address, std::string &str,
                              size_t length, size_t *count) {
-  char buf[length];
-  ErrorCode err = readMemory(ptid, address, buf, length, count);
-  if (err != kSuccess)
-    return err;
-
-  if (strnlen(buf, length) == length)
-    return kErrorNameTooLong;
-
-  str = std::string(buf);
-  return kSuccess;
+  DS2BUG("not implemented");
```

Comment:
I'd think that calling `PTrace::readString` on a Darwin build is a nasty bug, so something more specific than `"not implemented"` would be good. E.g.: `DS2BUG("impossible to use ptrace to %s on Darwin", __FUNCTION__);`